doc/whats-new.md

```diff
@@ -1,479 +1,206 @@
-and a column for the realization index. People parsing the CSV outputs will
-have to update their scripts.
+Prajacta Jadhav and Dharma Wijewickreme contributed the GMPE Youd et
+al. (2002) and the corresponding site parameters.
 
-In the single site case, there is now a warning when a realization
-does not contribute to the disaggregation. This helps identifying
-pathological situations.
+Tom Son contributed a few bug fixes to the GMPE Kuehn et al. (2020).
 
-We fixed a bug in the `Mag_Lon_Lat` exporter: the order of
-the columns was wrong and the fields mag, lon, lat were actually
-containing the values of lon, lat, mag.
+Finally, we raise an error when the total standard deviation is zero and the
+truncation level is not zero (currently this may happen only with the 
+GMPE YenierAtkinson2015BSSA).
 
-We added a command `oq info disagg` to print all 7 kinds of supported
-disaggregation outputs.
-
-We added a FAQ about how to compute mean disaggregation outputs, since
-many uses asked, see 
+# New hazard features
 
-https://github.com/gem/oq-engine/blob/engine-3.13/doc/faq-hazard.md#how-can-i-compute-mean-disaggregation-outputs
+The UCERF calculator has been removed. Now you can run an UCERF
+calculation by using the regular classical calculator, provided you
+have the UCERF model in a new format using multiFaultSources. The new
+format has substantial advantages, for instance it is possible to
+combine the UCERF model with other models.
 
-Finally we removed the long deprecated XML exporters.
+We implemented a cache for the distances that speedups
+substantially the calculation (we are talking about a 2x
+speedup). Since the distance calculation algorithm has changed
+internally, the numbers are expected to be slightly different than
+in past versions.
 
-# Improvements to the event based calculator
+If you already have a license for the UCERF model or you are a GEM
+sponsor you can request the new model for free.
 
-We significantly reduced the slow tasks affecting event based calculations,
-both for hazard and risk.
+At user request, it is now possible to disaggregate by all realizations
+by setting in the job.ini file
 
-We added a check for missing GSIMs in the job_risk.ini file, to
-avoid confusing error messages in the middle of the computation.
+`num_rlzs_disagg = 0`
 
-We extended the mag-dependent filtering to event based calculations; before
-it was honored only in classical calculations.
+Naturally, if you have a large number of realizations that can cause
+your calculation to run out of memory or to become extra-slow. Use
+this feature with care.
 
-The `custom_site_id` is now exported also by the GMF exporters.
+# New risk features
 
-When using the `--hc` option the engine was using the site collection
-of the parent calculation and ignoring the site collection of the
-child calculation: this is now fixed.
+At user request, we implemented multi-tag aggregation: it is now
```

Comment:
Maybe we can add the links to the issues / PRs: https://github.com/gem/oq-engine/issues/7663